cli/cli/commands/portal/start/start.go

```diff
@@ -22,32 +23,22 @@ var PortalStartCmd = &lowlevel.LowlevelKurtosisCommand{
 	PostValidationAndRunFunc: nil,
 }
 
-func run(ctx context.Context, _ *flags.ParsedFlags, args *args.ParsedArgs) error {
-	logrus.Infof("Starting Kurtosis Portal")
+func run(ctx context.Context, _ *flags.ParsedFlags, _ *args.ParsedArgs) error {
 	portalManager := portal_manager.NewPortalManager()
-
-	// Checking if new version is available and potentially downloading it
-	if _, err := portal_manager.DownloadLatestKurtosisPortalBinary(ctx); err != nil {
-		return stacktrace.Propagate(err, "An unexpected error occurred trying to download the latest version of Kurtosis Portal")
-	}
-
-	currentPortalPid, process, isPortalReachable, err := portalManager.CurrentStatus(ctx)
+	currentPortalPid, _, isPortalReachable, err := portalManager.CurrentStatus(ctx)
 	if err != nil {
 		return stacktrace.Propagate(err, "Unable to determine current state of Kurtosis Portal process")
 	}
+
 	if isPortalReachable {
 		logrus.Infof("Portal is currently running on PID '%d' and healthy.", currentPortalPid)
```

Comment:
Maybe add a comment here explaining why we're not trying to upgrade it (we don't want to break the currently running port forward, as the portal is stateless)